app/models/cell_metadatum.rb

```diff
@@ -61,7 +61,7 @@ def concatenate_data_arrays(array_name, array_type)
 
   # create dropdown menu value for annotation select
   def annotation_select_value
-    "#{self.name}--#{self.annotation_type}--study"
+    "#{self.name}--#{self.annotation_type}--#{can_visualize? ? 'study' : 'invalid'}"
```

Comment:
Ruby question - is the first `?` in `can_visualize? ? ...` used to convert `can_visualize` to a boolean?

Reply:
No, that's just the Ruby convention for naming a method that returns a boolean.  If you want to convert a value to boolean in Ruby, it's the standard preceding double-bang that many languages use, like so:
```
irb(main):001:0> true_val = true
=> true
irb(main):002:0> false_val = false
=> false
irb(main):003:0> nil_val = nil
=> nil
irb(main):004:0> !!true_val
=> true
irb(main):005:0> !!false_val
=> false
irb(main):006:0> !!nil_val
=> false
irb(main):007:0> string_val = 'foo'
=> "foo"
irb(main):008:0> !!string_val
=> true
```

Reply:
oh interesting, that will take some getting used to!😅 Thanks for the clarification!